Source/Scene/Model/Model.js

```diff
@@ -2095,7 +2096,11 @@ function scaleInPixels(positionWC, radius, frameState) {
 function getScale(model, modelMatrix, frameState) {
   let scale = model.scale;
 
-  if (model.minimumPixelSize !== 0.0 && !model._projectTo2D) {
+  if (
+    defined(frameState) &&
```

Comment:
This check feels a little awkward. Could `boundingSphere` and `computedScale` be updated separately? That way the `boundingSphere` setter would only need to update the bounding sphere which doesn't require `frameState`.